rockcraft/commands/init.py

```diff
@@ -80,6 +80,7 @@ class InitCommand(AppCommand):
             """\
                 name: {name}
                 base: ubuntu@22.04 # the base environment for this Flask application
+                build-base: ubuntu@22.04 # the build base environment for creating the OCI image
```

Comment:
"The build base environment for this Flask Application" no?

Reply:
You are right, probably I'll remove this again since it is only necessary if we had a bare base here or wanted it to be different, neither of which is needed

Reply:
Marking as resolved for now 🙂